examples/mapping_example.py

```diff
@@ -0,0 +1,113 @@
+#%%
+from math import isclose
+from pathlib import Path
+
+import numpy as np
+from mapping_functions import *
+
+
+def calculated_as_expected(expected, calculated):
+    ok = []
+    for i in range(len(expected)):
+        ok.append(isclose(expected[i], calculated[i], rel_tol=0.001, abs_tol=0.0))
+    return all(ok)
+
+
+workdir = Path(r"c:\src\imod_coupler\examples\input_example_mapping")
+
+# Test exchange MF-DFLOW1D
+# create dummy arrays to exchange
+
+# mf riv1-flux to exchange
+mf_riv1_flux = np.array([3, 3, 4, 4, 4])
+# dflow1d flux and stage to exchange
+dflow1d_flux = np.array([6, 7, 8])
+dflow1d_stage = np.array([4, 5, 6])
+
+# get dflow-id based on xy-coordinates after initialisation (now as test from file)
+dflow1d_lookup, ok_file = get_dflow1d_lookup(workdir)
```

Comment:
if ok_file is not used, you can make that explicit by renaming this variable to "_".
The underscore is commonly used for unused variables.